tests/integrators/assembler_simple_mgmatrix_04.cc

```diff
@@ -203,14 +204,14 @@ int main()
 
   // FESystem<2> sys1(p0, 2, p1, 1);
   // FESystem<2> sys2(p0, 2, rt0, 1);
-  // FESystem<2> sys3(rt0, 1, p0, 2);
+  FESystem<2> sys3(rt0, 1, p0, 1);
   // FESystem<2> sys4(p1, 2, q2, 2);
   // FESystem<2> sys5(q2, 2, p1, 2);
```

Comment:
I see. Do you think adding them adds any extra value, or are they duplicative to the other tests in this test case?